### ISSUE TITLE
[Feat]: I want to add my profile

### ISSUE THREAD

**Issue by vinitk1509** [codeaashu/DevDisplay#406]
### What feature?

I want to add my profile

### Add screenshots

Add screenshots
![image](https://github.com/user-attachments/assets/58af118e-d66b-4dc2-9a86-6b2587e305cf)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vinitk1509** [comment]
@codeaashu  please assign me

**Comment by ishap11** [comment]
Hi @vinitk1509 , thanks for your interest in contributing to DevDisplay!

Before we proceed, we noticed you adding your profile on #DevDisplay but forgot to star the repo. It's like making an amazing sandwich and forgetting to take a bite! -- I think our repo's feeling a little lonely without your 🤗 So, be sure to star the DevDisplay GitHub Repository. 🌟

💌 This is a mandatory step for all contributors. & It's is the first priority of contribution in DevDisplay 💌

🚀 We look forward to your contribution once these steps are completed. 🚀

Thanks 👍🏻
